packages/bitcore-node/migrations/fixUnspentInputs.js

```diff
@@ -0,0 +1,65 @@
+/****************************************
+*** This migration script will fix historical instances 
+*** of an issue where inputs that were used in an RBF tx
+*** are left in a pending state if they are not used by 
+*** the new tx that is the replacement.
+********************************************/ 
+
+const { TransactionModel, TransactionStorage } = require('../build/src/models/transaction');
+const { CoinModel, CoinStorage } = require('../build/src/models/coin');
+const { Storage } = require('../build/src/services/storage');
+const { wait } = require('../build/src/utils/wait');
+
+const RBFEnabledUTXOChains = [ 'BTC', 'LTC', 'DOGE' ];
+const networks = [ 'regtest', 'testnet', 'livenet'];
+const storageArgs = {
+    dbHost: '127.0.0.1',
+    dbName: 'bitcore'
+  };
+
+class Migration {
+    transactionModel = new TransactionModel();
+    coinModel = new CoinModel();
+
+    constructor({ transactionModel = TransactionStorage, coinModel = CoinStorage } = {}) {
+        this.transactionModel = transactionModel;
+        this.coinModel = coinModel;
+      }
+    async connect() {
+        try {
+            if (!Storage.connected) {
+                await Storage.start(storageArgs);
+                await wait(2000);
+              }
+        } catch (e) {
+            console.log(e);
+        }
+    }
+
+    async runScript() {
```

Comment:
Let's add a param to do a dry-run that outputs the txids that will be changed. By default, it should do a dry-run and then have a cli param to run it for real